adoc-mode.el

```diff
@@ -3783,6 +3787,17 @@ Turning on Adoc mode runs the normal hook `adoc-mode-hook'."
 ;;;; non-definitions evaluated during load
 (adoc-calc)
 
+
+;; Flyspell
+
+(defun adoc-flyspell-p ()
+  "Function for `flyspell-mode-predicate' property of `adoc-mode'.
+Returns t if word at point should be checked, nil otherwise."
+  (font-lock-ensure (line-beginning-position) (line-end-position))
+  (null (get-text-property (point) 'adoc-flyspell-ignore)))
```

Comment:
This `null` check also seems redundant.

Reply:
(I assume this will work just fine without explicit conversion into boolean, given that Elisp doesn't have a real boolean type anyways)

Reply:
Should be it be `t` specifically or just something truthy (anything that's not `nil`)? I'm asking mostly because I've rarely seen APIs that rely specifically on `t` as the return value.

Reply:
I misread the intention of `adoc-flyspell-p` - thought this was the things to ignore, but it's actually the things to check. That being said my initial remark was also because of some style conventions in Elisp:

> `null` is the same as `not`; both functions are included for the sake of clarity. As a matter of style, it is customary to use `null` to check whether something is the empty list and to use `not` to invert the sense of a logical value.

That's why I called `null` a predicate, despite its name. Anyways, not a big deal.

Reply:
> ... my initial remark was also because of some style conventions in Elisp:
> 
> > `null` is the same as `not`; both functions are included for the sake of clarity. As a matter of style, it is customary to use `null` to check whether something is the empty list and to use `not` to invert the sense of a logical value.

Good point. I've changed it.